README.md

```diff
@@ -36,6 +36,12 @@ Options:
 - run a set of files to create the baseline data, e.g. `xterm-benchmark some_files* -b`
 - after code changes run the same set of files against the baseline, e.g. `xterm-benchmark some_files* -e`
 
+### Running Example Perf Case
+- cd xterm-benchmark && npm install (If this throws puppeteer error then try deleting the whole puppeteer folder before doing npm install again.)
+- npm run tsc ( Compiles ts files into js files which can be found under ./lib folder)
+- node lib/cli.js lib/src/xterm_perfcases/parser.ts ( New perf case files should be under src/xterm_perfcases )

```

Comment:
The target path does not contain `src` and the file ending should be `.js`

Reply:
@jerch , sorry for that, it was a mistake from my end.
Changed it now :)